### PR TITLE
Connect to Rabbit

### PR DESCRIPTION
**Pull request by beckje01**
Add docker-compose for local dev.
Use circles rabbit for testing

### CODE REVIEW

File: ratpack-rabbit-producer/src/test/groovy/smartthings/ratpack/rabbit/RabbitConnectionServiceSpec.groovy [hunk 1]
```diff
@@ -13,6 +13,10 @@ class RabbitConnectionServiceSpec extends Specification {
 	@AutoCleanup
 	ExecHarness harness = ExecHarness.harness()
 
+	String getTestRabbitServer() {
+		return System.getenv("RABBIT_SERVER") ?: '192.168.99.100'
```
Comment (llinder) [inline ratpack-rabbit-producer/src/test/groovy/smartthings/ratpack/rabbit/RabbitConnectionServiceSpec.groovy]: I would prefer defaulting to localhost.  Given that docker on Linux and both release candidates for Mac and Windows bind to localhost it seems like a better default.  If thats not desirable for some then a custom domain name might be a good alternative.

Comment (beckje01) [inline ratpack-rabbit-producer/src/test/groovy/smartthings/ratpack/rabbit/RabbitConnectionServiceSpec.groovy]: I'm ok changing to that if we think everyone is moving to that.

**Comment by Bijnagte** [comment]
Looks good to me outside of @llinder 's comment

**Comment by llinder** [comment]
👍

**Comment by minzhang28** [comment]
Hi Jeff,
Some comments for the docker-compose file.
Is it possible for you to update the docker-compose file to V2? Your compose file is in V1 format, which will be deprecated, also it only supports Compose up to 1.6.x. It's not affecting anything if you can run this on all your environment, but there will be a moment, we failed to run.

For V2 format, it's quite easy, please refer to 
https://docs.docker.com/compose/compose-file/#version-2

Thanks
Min

**Pull request merged**